task_23/dima.gres/src/main/java/Main.java

```diff
@@ -0,0 +1,25 @@
+import java.util.ArrayList;
+
+public class Main {
```

Comment:
Let's always use package. It is not good to put files in the default (none) package.